samples/openapi3/client/petstore/python/tests/test_model.py

```diff
@@ -408,7 +408,12 @@ def test_valdiator(self):
         self.assertEqual(a.pattern_with_digits_and_delimiter, "image_123")
 
         # test sanitize for serializaation with SecretStr (format: password)
-        self.assertEquals(petstore_api.ApiClient().sanitize_for_serialization(a), {'byte': b'string', 'date': '2013-09-17', 'number': 123.45, 'password': 'testing09876', 'pattern_with_digits_and_delimiter': 'image_123'})
```

Comment:
thanks for adding another test below

shouldn't this test/assertion still pass after this change?

Reply:
> shouldn't this test/assertion still pass after this change?

yes, you are right... it can make sense to keep this line also as in case of password which will be as default string - it will also work. I added this line back to this pull.. https://github.com/OpenAPITools/openapi-generator/pull/18023/commits/0ed3ba487e27dd5a7b97e417f7fc865141f12d18 + will check sanitize function based on bytes/date/integer data types

Reply:
@wing328  - can you please check and confirm that now is ok?